src/index.html

```diff
@@ -1,13 +1,63 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <section class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="Apple iMac 27 inch. Retina 5K Monoblock"
+        class="card__image"
+      >
+      <h1 class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <div class="card__code">
+        Product code: 195434
+      </div>
+
+      <div class="stars">
+        <ul class="stars__rating">
+          <li class="stars__star"></li>
+
+          <li class="stars__star"></li>
+
+          <li class="stars__star"></li>
+
+          <li class="stars__star"></li>
+
+          <li class="stars__star"></li>
+        </ul>
+
+        <div class="stars__reviews">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price-text">Price:</div>
+        <div class="card__price-amount">$2,199</div>
+      </div>
+
+      <a href="#"
+        class="card__buy-button"
+        data-qa="hover">
```

Comment:
Closed bracket should be on same line as open
```suggestion
      <a href="#"
        class="card__buy-button"
        data-qa="hover"
      >
```